app/models/repository.rb

```diff
@@ -20,6 +20,10 @@ def self.license_options
      "Attribution - Non-Commercial - No Derivatives"]
   end
 
+  def user_username
```

Comment:
Have you looked into removing this method and fixing the tests? #58 is to remove the column